content/blog/intro-to-step-functions/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Intro to AWS Serverless Step Functions"
+date: 2020-02-13
+meta_desc: "An Introduction to building serverless apps with AWS Step Functions."
+meta_image: meta-step-function.png
+authors:
+    - sophia-parafina
+tags:
+    - serverless
+    - "step functions"
```

Comment:
Not 100% sure if this needs to be "step-functions" or not.

On a related not, it looks like we have `f`, `c`, and `cd` as blog tags. Which I assume are from other posts where the tags didn't get captured/interpreted correctly. Would you mind fixing that? Maybe we need to add some checks in the Hugo build to report an error if we have unexpected tags in the future?

Reply:
AWS has it as 'Step Functions' so I went with that.

We've discussed cleaning up the tags, but are afraid that it will break links. It's on the radar but on the back burner.